diff --git a/lib/config.js b/lib/config.js
index 6432edccd969d41584c7245b5c0c5f0dce6a8d52..f9ebba10605dd63435dd69f1db057a25691d75b0 100644
--- a/lib/config.js
+++ b/lib/config.js
@@ -1,6 +1,7 @@
 import { __awaiter } from "tslib";
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
+import { nodeTypes } from '@mdx-js/mdx';
 import { alias } from '@pkgr/es-modules';
 import { DEV, EXTENSIONS, NODE_MODULES_REG, PROD, __DEV__, __PROD__, findUp, identify, isAngularAvailable, isPkgAvailable, isReactAvailable, isSvelteAvailable, isTsAvailable, isVueAvailable, tryExtensions, tryFile, tryPkg, tryRequirePkg, } from '@pkgr/utils';
 import FriendlyErrorsWebpackPlugin from '@soda/friendly-errors-webpack-plugin';
@@ -219,6 +220,7 @@ export default ({ entry = 'src', outputDir = 'dist', externals, type, copies = [
                             loader: '@mdx-js/loader',
                             options: {
                                 remarkPlugins: [(yield import('remark-gfm')).default],
+                                rehypePlugins: [(yield import('rehype-raw')).default, {passThrough: nodeTypes}],
                             },
                         },
                     ],